examples/05-nautobot-peeringdb/adapter_nautobot.py

```diff
@@ -0,0 +1,201 @@
+"""Diffsync adapter class for Nautobot."""
+import os
+import requests
+from diffsync import DiffSync
+
+from .models import RegionModel, SiteModel
+
+NAUTOBOT_URL = os.getenv("NAUTOBOT_URL", "https://demo.nautobot.com")
+NAUTOBOT_TOKEN = os.getenv("NAUTOBOT_TOKEN", "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
+
+
+class RegionNautobotModel(RegionModel):
+    """Implementation of Region create/update/delete methods for updating remote Nautobot data."""
+
+    @classmethod
+    def create(cls, diffsync, ids, attrs):
+        """Create a new Region record in remote Nautobot.
+        Args:
+            diffsync (NautobotRemote): DiffSync adapter owning this Region
+            ids (dict): Initial values for this model's _identifiers
+            attrs (dict): Initial values for this model's _attributes
+        """
+        data = {
+            "name": ids["name"],
+            "slug": attrs["slug"],
+        }
+        if attrs["description"]:
+            data["description"] = attrs["description"]
+        if attrs["parent_name"]:
+            data["parent"] = str(diffsync.get(diffsync.region, attrs["parent_name"]).pk)
```

Comment:
Could we also do:

```python
data["parent"] = {"name": attrs["parent_name"]}
```

as in the `update` case below? Or is there a reason for the different approach between these two methods?

Reply:
I only tried the `create` for this model... so the `update` was not updated accordingly, will do.